packaging/docker/Dockerfile.ubuntu

```diff
@@ -0,0 +1,56 @@
+ARG BASE_IMAGE=ubuntu:18.10
+FROM ${BASE_IMAGE} AS grafana-builder
+
+ARG GRAFANA_TGZ="grafana-latest.linux-x64.tar.gz"
+
+COPY ${GRAFANA_TGZ} /tmp/grafana.tar.gz
+
+RUN mkdir /tmp/grafana && tar xfz /tmp/grafana.tar.gz --strip-components=1 -C /tmp/grafana
+
+FROM ${BASE_IMAGE}
+
+EXPOSE 3000
+
+ARG GF_UID="472"
+ARG GF_GID="472"
+
+ENV PATH=/usr/share/grafana/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
+    GF_PATHS_CONFIG="/etc/grafana/grafana.ini" \
+    GF_PATHS_DATA="/var/lib/grafana" \
+    GF_PATHS_HOME="/usr/share/grafana" \
+    GF_PATHS_LOGS="/var/log/grafana" \
+    GF_PATHS_PLUGINS="/var/lib/grafana/plugins" \
+    GF_PATHS_PROVISIONING="/etc/grafana/provisioning"
+
+WORKDIR $GF_PATHS_HOME
+
+# We need curl in image
```

Comment:
@xlson Aha, I guess I didn't realize/forgot that PhantomJS is in the tarball. Can we avoid installing font libs on other architectures than x86-64 though, since they are only needed by PhantomJS?

Reply:
Yeah, why not.